openhands/cli/main.py

```diff
@@ -304,6 +308,20 @@ def on_event(event: Event) -> None:
         )
 
         runtime.config.mcp.stdio_servers.extend(openhands_mcp_stdio_servers)
+        # Add `fetch` MCP server by default if not already present and `uvx` is available
+        if shutil.which('uvx') is not None:
+            fetch_mcp_config = MCPStdioServerConfig(
+                name='fetch',
+                command='uvx',
+                args=['mcp-server-fetch'],
+            )
+            if fetch_mcp_config not in runtime.config.mcp.stdio_servers:
+                runtime.config.mcp.stdio_servers.append(fetch_mcp_config)
+        else:
+            logger.warning(
+                'uvx is not installed or not found in PATH. '
+                'The fetch MCP server will be disabled.'
+            )
```

Comment:
Since you add the check in openhands/mcp/utils.py, i don't think we need this anymore?